### ISSUE TITLE
Add Notes: Error in reading tags

### ISSUE THREAD

**Issue by tghoward** [geopaparazzi/geopaparazzi#665]
A fresh install of Geopap from the play store gives me this error ("Error in reading tags") when trying to add a new note. 

I have tried putting a tags.json file in these places on the device:
//Android/data/eu.hydrologis.geopaparazzi/
//Android/data/eu.hydrologis.geopaparazzi/files
//geopaparazzi

but with no luck. No matter what, I can't get geopap to read the tags.json file. I'm guessing this is related to new Android permissions?
![Screenshot_20230331-121307](https://user-images.githubusercontent.com/6129193/229176340-4449c195-7d84-4de5-8595-defc6b7be7c5.png)

I'm attaching a screenshot of the log file. That suggests it's a null-pointer error. 

Any suggestions?
Thanks,
Tim

**Comment by moovida** [comment]
Dear Tim, I just gave it a try on my device with android 13 and everything works as expected. 
It should not suffer of all the permission issues SMASH suffers from, because it was built before all the mess started and is kept working in compatibility mode by the play store. 

Is it possible that it is reading a form it is not able to decode? Did you try to remove all forms and see if the default ones work?

**Comment by tghoward** [comment]
That's good to hear that you have no issues with Android 13. I was testing (and getting the same results) on an Android 12 device and an Android 9 device. 

Yes, I was using the default form that comes with Geopaparazzi. I'll try a thorough file removal after uninstalling and see if that helps.

**Comment by tghoward** [comment]
I just tried this (Android 12 Moto g100) 
1. uninstall geopap
2. Delete geopaparazzi folder at main storage, remove any .gpap files. Files in ~/Android/data were removed from uninstall.
3. reboot. Re-install geopaparazzi using Play Store. 
4. it looks like the only folder/files that get installed visible to the user are at ~/Android/data/eu.hydrologis.geopaparazzi
5. Fire up geopap. It makes a default .gpap file (and journal file) at the user's root. It kept some of my settings (weird?). 
6. Default forms still don't work. 

But, since it works for you, I'll assume it's something on my end and I'll test a few more units.   We can close this for now -- you know there may be something going on with forms on some units ...
